src/index.js

```diff
@@ -107,6 +99,11 @@ function API(workspaceId, impactUrl='') {
 }
 
 // Private methods ///////////////////////////////////////////////////////////
+API.prototype._request = function(path, method, init) {
+    const impactUrl = this._impactUrl; // Cannot reference 'this' in callback!
```

Comment:
The comment is obsolete or incorrect I guess?

Reply:
No? However, I modified the code such that this comment is not needed for clarification.